specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -321,6 +321,7 @@ scope kafka
 
         struct KafkaGroupBeginEx
         {
+            string16 topic;
```

Comment:
Why do we need `topic` on `KafkaGroupBeginEx`?
Isn't the consumer group independent of any specific topic?